js/preload/preloadMain.js

```diff
@@ -86,6 +86,20 @@ const throttledSetIsInMeetingStatus = throttle(1000, function (isInMeeting) {
 local.ipcRenderer.on('on-page-load', () => {
     snackBar = new SnackBar();
 
+    // Enable spellchecker only for main window until
+    // the underline memory leak is fixed on electron
+    // https://github.com/electron/electron/issues/15459
+    if (window.name === 'main') {
+        webFrame.setSpellCheckProvider('en-US', true, {
+            spellCheck (text) {
+                return !local.ipcRenderer.sendSync(apiName, {
```

Comment:
anyway we can change this async to avoid blocking of the main thread?

Reply:
I don't think this is possible yet. https://github.com/electron/electron/issues/2065